content/blog/2019/2019-09-28-on-expected-vomiting-time.md

```diff
@@ -0,0 +1,90 @@
++++
+author="Janez Demšar"
+date= '2019-09-28'
+draft= true
+title="On Expected Vomiting Time"
+type="blog"
+thumbImage = "/blog_img/2019-09-28-vomiting-pumpkin.jpg"
```

Comment:
You have missed one folder level. It should be /blog_img/**2019**/2019-09-28-vomiting-pumpkin.jpg.
You can preview your blog by running `hugo serve`.